metadeploy/api/models.py

```diff
@@ -63,9 +63,17 @@ class Meta:
     id = HashidAutoField(primary_key=True)
 
 
+class MarkdownField(BaseMarkdownField):
+    def __init__(self, *args, **kwargs):
+        kwargs["property_suffix"] = kwargs.get("property_suffix", "_markdown")
+        kwargs["blank"] = kwargs.get("blank", True)
+        kwargs["help_text"] = kwargs.get("help_text", "Markdown is supported")
+        super().__init__(*args, **kwargs)
```

Comment:
I would have preferred to make these changes in sfdo-template-helpers rather than introducing a subclass here.

Reply:
Makes sense, we can do that 👍

Reply:
@jgerigmeyer At this point, don't bother. But in the future if you want to modify something from sfdo-template-helpers, let's discuss if it would make more sense to do it there.

Reply:
@davisagli Okay, if you prefer that. I'm not sure what "at this point" is here, or why we wouldn't bother -- this was a 5 minute change, plus one data migration. I guess my default for something like this is to make the change, see how it looks (how much code it touches, how it works), *then* abstract it to a helper app if everything looks good.

Reply:
I just have a lot of things in flight and don't want to add reviewing this change in sfdo-template-helpers (and any implications for other apps using the helpers, and getting them updated) today.

Reply:
@davisagli Makes sense. It's not time-sensitive -- I may open a PR in template-helpers this week, but it wouldn't need to be reviewed/merged right away.

Reply:
No rush, but a PR is in: https://github.com/SFDO-Tooling/sfdo-template-helpers/pull/37

I still think the subclass in MD makes sense, since we're consistently overriding some of the defaults (`property_suffix` and `blank`), but I moved the `help_text` default into sfdo-template-helpers.